app/views/admin/hackathons/show.html.erb

```diff
@@ -27,7 +27,7 @@
         <% if @hackathon.banner.present? %>
           <div>
             <h2 class="heading--small">Banner</h2>
-            <%= image_tag @hackathon.banner, height: 128, style: "background-color: black;" %>
+            <%= image_tag @hackathon.banner.variant(:small), height: 128 %>
```

Comment:
I think we should show the whole image here

Reply:
Why? It's the same aspect ratio as other places.

Reply:
If the banner submitted has a 3:4 aspect ratio, a portion will be cropped. Because we have human reviewers making sure that the banner meets our standards (e.g. doesn't contain text), I want them to be able to see the uncropped image. Otherwise, they'd really only be approving the portion of the image that fits inside the 16:9. This is important in the case we decide to use the banners in another aspect ratio.

Here is a hypothetical example:
![Untitled (1)](https://github.com/hackclub/hackathons-backend/assets/20099646/a4465338-2116-4ed6-adbd-931cc3be2809)
With this 3:4 image, the reviewer would only see the red 16:9 portion. The red portion looks good and they'd approve it. In the future, we decide to create a "High School Hackathons Wrapped" and use this banner; only to later realize that it contains text

Reply:
One solution besides showing the full image is to create a variant with `resize_to_limit`

https://github.com/janko/image_processing/blob/master/doc/vips.md#resize_to_fit

Reply:
That's what the small variant is, it isn't cropping anything.